core/src/main/java/tc/oc/pgm/flag/post/SinglePost.java

```diff
@@ -0,0 +1,187 @@
+package tc.oc.pgm.flag.post;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import com.google.common.collect.ImmutableList;
+import java.time.Duration;
+import java.util.Random;
+import javax.annotation.Nullable;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import tc.oc.pgm.api.feature.FeatureReference;
+import tc.oc.pgm.api.filter.Filter;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.flag.Flag;
+import tc.oc.pgm.points.AngleProvider;
+import tc.oc.pgm.points.PointProvider;
+import tc.oc.pgm.points.PointProviderLocation;
+import tc.oc.pgm.teams.TeamFactory;
+
+public class SinglePost extends PostDefinition {
```

Comment:
Why not call this just `Post`?

Reply:
Post is the match-time Post, this is the map-time "PostDefinition" (for a single post)